lib/DDG/Spice/Wikidata.pm

```diff
@@ -0,0 +1,32 @@
+package DDG::Spice::Wikidata;
+# ABSTRACT: Uses Wikidata to answer simple questions
+
+use strict;
+use DDG::Spice;
+use Text::Trim;
+
+name "Wikidata";
+source "Wikidata";
+icon_url "https://www.wikidata.org/static/favicon/wikidata.ico";
+description "Answer simple questions for properties on subjects";
+primary_example_queries "population of Germany", "president of the United States";
```

Comment:
Can we find more unique examples? Both these queries are already covered by existing IA's. Would like to use this one to fill more niche gaps if possible